tests/ClientTest.php

```diff
@@ -64,6 +64,42 @@ public function registerShutdownFunction()
 
 class ClientTest extends TestCase
 {
+    /**
+     * @dataProvider constructorInitializesTransactionStackDataProvider
+     */
+    public function testConstructorInitializesTransactionStack($isHttpRequest)
+    {
+        if ($isHttpRequest) {
+            $_SERVER['PATH_INFO'] = '/foo';
```

Comment:
Do we have the "backup globals" option enabled in the PHPUnit config?

Reply:
Yes we do

https://github.com/getsentry/sentry-php/blob/2841bb392de677c15497737e3068d00b36d923ce/phpunit.xml.dist#L6